app/code/Academy/SmsSubscription/Plugin/AddAttributesToUiDataProvider.php

```diff
@@ -0,0 +1,27 @@
+<?php
+declare(strict_types=1);
+
+namespace Academy\SmsSubscription\Plugin;
+
+use Academy\SmsSubscription\Ui\DataProvider\Subscription\ListingDataProvider as SubscriptionDataProvider;
+use Magento\Framework\View\Element\UiComponent\DataProvider\SearchResult;
+
+class AddAttributesToUiDataProvider
+{
+    public function afterGetSearchResult(SubscriptionDataProvider $subject, SearchResult $result): SearchResult
+    {
+        if ($result->isLoaded()) {
+            return $result;
+        }
+
+        $column = 'entity_id';
```

Comment:
Is it really necessary to create a variable for it considering it's being used in only one place a few lines below?